conf/app.profile.routes

```diff
@@ -0,0 +1,96 @@
+GET         /create-profile/start                                      controllers.profile.ProfileSetupController.onPageLoad()
+POST        /create-profile/start                                      controllers.profile.ProfileSetupController.onSubmit()
+
+GET         /create-profile/check-your-answers                         controllers.profile.CyaCreateProfileController.onPageLoad()
+POST        /create-profile/check-your-answers                         controllers.profile.CyaCreateProfileController.onSubmit()
+
+GET         /create-profile/ukims-number                               controllers.profile.UkimsNumberController.onPageLoadCreate(mode: Mode = NormalMode)
+POST        /create-profile/ukims-number                               controllers.profile.UkimsNumberController.onSubmitCreate(mode: Mode = NormalMode)
+
+GET         /update-profile/ukims-number                               controllers.profile.UkimsNumberController.onPageLoadUpdate(mode: Mode = NormalMode)
+POST        /update-profile/ukims-number                               controllers.profile.UkimsNumberController.onSubmitUpdate(mode: Mode = NormalMode)
+
+GET         /new/ukims-number                                          controllers.profile.NewUkimsNumberController.onPageLoad(mode: Mode = NormalMode)
```

Comment:
i think the new ukims stuff should be in a separate package - i know they are related but i think it's its own journey that shouldn't have any crossover

Reply:
Our thinking was It is part of the profile journey. At the moment we were just looking at making child classes of the root level to make the code tidier/more readable etc. Perhaps a further iteration could be to look into splitting these into separate nested packages, in this case within profile, if we believe that this is worthwhile.